Dockerfile

```diff
@@ -32,5 +32,5 @@ RUN pip3 install .
 WORKDIR /counterparty-cli
 RUN pip3 install .
 
-ENTRYPOINT [ "counterparty-server", "start" ]
+ENTRYPOINT [ "counterparty-server"]
 CMD [ "-h" ]
```

Comment:
Do we want the CMD not to be `start`?

Reply:
you should just be able to do `docker run` and have the service start

Reply:
wait, is `start` the default now?

Reply:
There is no default command for now. The `start` was removed to allow people to use the docker image for something other than `start` and without using the `--entrypoint` flag. It is not standard to include a command in the entrypoint.. this is what the CMD directive is intended for.

Reply:
hm, I don't follow... so if we set `CMD` to `start`, then wouldn't users be able to do something other than `start` without changing `ENTRYPOINT`? (isn't that the point of having `CMD`? 😅)

Reply:
yes we say the same thing :-)) I misunderstood you sorry 😅